src/App.tsx

```diff
@@ -15,71 +16,93 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selected: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.PureComponent<{}, State> {
+  state: Readonly<State> = {
+    selected: 'Jam',
```

Comment:
<img width="506" alt="Screenshot 2022-09-16 at 18 14 22" src="https://user-images.githubusercontent.com/47606379/190672572-e2572bda-165d-4a62-923e-e8f3cf115d5a.png">

```suggestion
    selectedGood: 'Jam',
```